src/CookieBanner/CookieBanner.vue

```diff
@@ -91,41 +93,61 @@ export default {
   box-shadow: 0px 24px 40px rgba(22, 25, 49, 0.1),
     0px 10px 16px rgba(22, 25, 49, 0.08), 0px 1px 0px rgba(22, 25, 49, 0.05);
   position: fixed;
-  bottom: 1.5rem;
+  bottom: 2rem;
   left: 50%;
   transform: translateX(-50%);
   z-index: 3000;
-  width: 100%;
-  max-width: 26.8125rem;
   background: #ffffff;
   border-width: 0.0625rem;
   border-style: solid;
   border-color: transparent;
   border-image: initial;
-  border-radius: 3.4375rem;
-  padding: 0.75rem 1.4375rem 0.75rem 1.4375rem;
+  border-radius: 0.5rem;
 }
 
 .wrapper {
   display: flex;
-  -webkit-box-align: center;
-  align-items: center;
-  -webkit-box-pack: justify;
-  justify-content: space-between;
 }
 
 .message {
-  font-size: 0.85rem;
+  font-size: 0.8125rem;
+  line-height: 1.125rem;
+  letter-spacing: 0.01em;
+  align-items: stretch;
+  justify-content: center;
+  padding: 0.8125rem;
```

Comment:
I think we can remove the right padding here, since there's already whitespace in the button itself. 👍

Reply:
https://github.com/tendermint/cosmos-ui/pull/144/commits/ca7f5a304bf00034d18017c4b1f6ec6c6eb0f29a